rpa-bot/1.6/modules/ROOT/pages/_partials/hard-soft-requirements.adoc

```diff
@@ -1,6 +1,9 @@
 == Hardware Requirements
 
 * For optimal performance, use a dedicated computer for running the RPA Bot.
+* Use a computer with the required amount of RAM. The amount of RAM required depends on the number of processes to run in parallel.
+
```

Comment:
If you want to make it a hanging paragraph, add `--` after the `+` on line 5 and a closing `--` after the `++++` on line 32.